### ISSUE TITLE
Feature request for AOR Reports:  Trimming multi-line field, use jquery tool-tip to show full content on mouse-over

### ISSUE THREAD

**Issue by rholighaus** [salesagility/SuiteCRM#1410]
It would make reports much more readable if we could configure a multi-line field (e.g. description) to be cut-off (and limited to one line!) in a report with a mouse-over showing the full field content in a e.g. a tool-tip when moving the mouse over the field.

This would be extremely helpful especially in multi-line fields that would otherwise make a report line to become to high.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature request for AOR Reports: Trimming multi-line field, use jquery tool-tip to show full content on mouse-over](https://trello.com/c/zDox9iQp/97-feature-request-for-aor-reports-trimming-multi-line-field-use-jquery-tool-tip-to-show-full-content-on-mouse-over)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
